pygmo/_island_test.py

```diff
@@ -275,10 +275,17 @@ def run_basic_tests(self):
         from . import mp_island
         from copy import copy, deepcopy
         from pickle import dumps, loads
+        # Try shutting down a few times.
```

Comment:
Maybe just add some explanation on why ... it looks weird to have these lines repeated one may think its just a typo. (same below.